dpdk/vr_dpdk_ethdev.c

```diff
@@ -162,6 +162,27 @@ vr_dpdk_ethdev_ready_queue_id_get(struct vr_interface *vif)
     return VR_DPDK_INVALID_QUEUE_ID;
 }
 
+/* Release ethdev RX queue */
+static void
+dpdk_ethdev_rx_queue_release(unsigned lcore_id, struct vr_interface *vif)
```

Comment:
If the number of RX queues is less than the number of lcores, will this function be called on those lcores which do not have a RX queue assigned to them?